golang/internal/config/config.go

```diff
@@ -19,16 +25,23 @@ type CommonConfiguration struct {
 	DefaultTimeout       time.Duration `yaml:"defaultTimeout"        env:"DEFAULT_TIMEOUT"         env-default:"5s"`
 	GrpcKeepalive        time.Duration `yaml:"grpcKeepalive"         env:"GRPC_KEEPALIVE"          env-default:"60s"`
 	Debug                bool          `yaml:"debug"                 env:"DEBUG"                   env-default:"false"`
-	GrpcInsecure         bool          `yaml:"grpcInsecure"          env:"GRPC_INSECURE"           env-default:"false"`
-	ImportContainerImage string        `yaml:"importContainerImage"  env:"IMPORT_CONTAINER_IMAGE"  env-default:"rclone/rclone:1.57.0"`
-	IngressRootDomain    string        `yaml:"ingressRootDomain"     env:"INGRESS_ROOT_DOMAIN"     env-default:""`
-	// TODO(c3ppc3pp): custom UUIDv4 setter
-	NodeID            string         `yaml:"nodeID" env:"NODE_ID" env-default:"cb7e9573-9a43-4d5b-8005-eb8bb7a423c4"`
-	ReadHeaderTimeout time.Duration  `yaml:"readHeaderTimeout"    env:"READ_HEADER_TIMEOUT"      env-default:"15s"`
-	Registry          string         `yaml:"registry"             env:"REGISTRY"                 env-default:"index.docker.io"`
-	RegistryPassword  string         `yaml:"registryPassword"     env:"REGISTRY_PASSWORD"        env-default:""`
-	RegistryUsername  string         `yaml:"registryUsername"     env:"REGISTRY_USERNAME"        env-default:""`
-	SecretPrivateKey  ConfigFromFile `yaml:"secretPrivateKeyFile" env:"SECRET_PRIVATE_KEY_FILE"  env-default:"/srv/dagent/private.key"`
-	// GRPC token is set separately, because nested structures are not yet suppported in cleanenv
-	GrpcToken *ValidJWT
+	// TODO(c3ppc3pp): custom setter to validate input
+	GrpcToken            string         `yaml:"grpcToken"             env:"GRPC_TOKEN"`
```

Comment:
It seems that you have the wrong set of changes in your changes, the deleted ones are the valid ones to keep except the custom type of UUID, but make sure to remove the UUIDv4 comment.

Reply:
oops, my bad. I'll fix it in a jiffy

Reply:
> except the custom type of UUID

Excuse me for not understanding, but should I also remove the custom UUID type?
Because I have used it for SetValue()...

Reply:
But the order is still messed up, you should not have anything else changed, but the custom type, the new type is good as it is right now.

Reply:
Okay, GrpcToken should be in the end, right?

Also, will it affect behavior?

Reply:
> Okay, GrpcToken should be in the end, right?
Yes.
> Also, will it affect behavior?
The behavior is not changed, the git history is cleaner and better representing what is going on.

Reply:
> Yes.

Done😀